specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/computeRPCommon.json

```diff
@@ -998,7 +1000,7 @@
         },
         "createOption": {
           "$ref": "#/definitions/CreateOption",
-          "description": "Specifies how the virtual machine should be created. Possible values are: **Attach.** This value is used when you are using a specialized disk to create the virtual machine. **FromImage.** This value is used when you are using an image to create the virtual machine. If you are using a platform image, you should also use the imageReference element described above. If you are using a marketplace image, you should also use the plan element previously described."
+          "description": "Specifies how the virtual machine disk should be created. Possible values are<br>**Attach:** This value is used when you are using a specialized disk to create the virtual machine.<br>**FromImage:** This value is used when you are using an image to create the virtual machine data disk. If you are using a platform image, you should also use the imageReference element described above. If you are using a marketplace image, you should also use the plan element previously described. <br>**Empty:** This value is used when creating an empty data disk. <br>**Copy:** This value is used to create a data disk from a snapshot.<br>**Restore:** This value is used to create a data disk from a disk restore point."
```

Comment:
we no longer use html tags like <br> as they shows up as raw text in most documentation. Please remove them here.